docs/tutorial/get-started.md

```diff
@@ -4,27 +4,42 @@ Vald is a highly scalable distributed fast approximate nearest neighbor dense ve
 Vald is designed and implemented based on Cloud-Native architecture.
 
 This article will show you how to deploy and run the Vald components on your kubernetes cluster.
-Fashion-mnist is used as an example of a dataset.
+And, Fashion-mnist is used as an example of a dataset.
+
+## Overview
+
+Before starting, let's check the below image.
+It shows the architecture image about the deployment result of Get Started.<br>
+The 4 kinds of microservices, `Vald LB Gateway`, `Vald Discoverer`, `Vald Agent` and `Vald Index Manager` will deployed on kuberntes.
+
+<img src="../../assets/docs/tutorial/getstarted.png" />
```

Comment:
Should we add some brief explanation for these components? 🤔

Reply:
IMO, it is not the responsibility of this document.
I think it is better to link any document when we create